### PR TITLE
Update RepeatMasker tutorial

### PR DESCRIPTION
**Pull request by jkreplak**
- Add a new tool to mask genome ab-initio, RED. 
- Add two new custom TE libraries for RepeatMasker

### CODE REVIEW

File: topics/genome-annotation/tutorials/repeatmasker/tutorial.md [hunk 1]
```diff
@@ -1,43 +1,35 @@
...
-  - It is an essential first step before running structural annotation pipelines
+  - RepeatMasker can be used to soft-mask a genome  - It is an essential first step before running structural annotation pipelines
 contributors:
-  - abretaud
-  - alexcorm
-  - lleroi
-  - r1corre
-  - stephanierobin
-  - erasmusplus
+  - abretaud  - alexcorm  - lleroi  - r1corre  - stephanierobin  - erasmusplus  - jkreplak
```
Comment (bgruening) [inline topics/genome-annotation/tutorials/repeatmasker/tutorial.md:18]: this looks wrong

File: topics/genome-annotation/tutorials/repeatmasker/tutorial.md [hunk 2]
```diff
@@ -1,43 +1,35 @@
...
 
 abbreviations:
-    SINEs: Short Interspersed Nuclear Elements
-    LINEs: Long Interspersed Nuclear Elements
+    SINEs: Short Interspersed Nuclear Elements    LINEs: Long Interspersed Nuclear Elements
```
Comment (bgruening) [inline topics/genome-annotation/tutorials/repeatmasker/tutorial.md:21]: this as well

File: topics/genome-annotation/tutorials/repeatmasker/tutorial.md [hunk 3]
```diff
@@ -1,43 +1,35 @@
...
 
 follow_up_training:
 - type: "internal"
-  topic_name: genome-annotation
-  tutorials:
+  topic_name: genome-annotation  tutorials:
```
Comment (bgruening) [inline topics/genome-annotation/tutorials/repeatmasker/tutorial.md:25]: something is wrong with your linebreaks :(

File: topics/genome-annotation/tutorials/repeatmasker/tutorial.md [hunk 4]
```diff
@@ -1,30 +1,30 @@
----
```
Comment (abretaud) [inline topics/genome-annotation/tutorials/repeatmasker/tutorial.md:1]: no need to change this

File: topics/genome-annotation/tutorials/repeatmasker/tutorial.md [hunk 5]
```diff
@@ -34,10 +34,10 @@ follow_up_training:
     - funannotate
 subtopic: eukaryote
 priority: 1
----
```
Comment (abretaud) [inline topics/genome-annotation/tutorials/repeatmasker/tutorial.md:37]: no need to change this

**Review by abretaud — COMMENTED** [review]
Merci Jonathan! some changes to do in the header

File: topics/genome-annotation/tutorials/repeatmasker/tutorial.md [hunk 6]
```diff
@@ -22,10 +22,11 @@ contributors:
   - r1corre
   - stephanierobin
   - erasmusplus
+  - jkreplak
```
Comment (abretaud) [inline topics/genome-annotation/tutorials/repeatmasker/tutorial.md:25]: You should add yourself in the `CONTRIBUTORS.yaml` file at the root of the repo

**Comment by abretaud** [comment]
@jkreplak I've just pushed a few commits to fix the syntax here and there + adjust a few phrases, can you have a look?
Also you need to add yourself as contributor (see my other comment)

File: topics/genome-annotation/tutorials/repeatmasker/tutorial.md (hunk 6 above)
Comment (jkreplak) [reply topics/genome-annotation/tutorials/repeatmasker/tutorial.md:25]: Done !

**Comment by jkreplak** [comment]
Thanks, @abretaud I've just reviewed your changes, seems great for me.

**Review by abretaud — APPROVED** [review]

**Pull request merged**